exporter/metrics_exporter.go

```diff
@@ -141,19 +147,35 @@ func (m *metricsExporter) recordMetrics() {
 		// concurrently, so we assume that as long as that cap isn't enormous (which
 		// would only occur on an enormous cluster), it's practical to iterate over
 		// all the running workers.
+		var countError error
```

Comment:
Is there a reason we're not just reusing the existing `err` variable?

Reply:
I mentioned in a separate thread that I didn't want to set `m.currentRunningJobs` if there was an error inside the loop, so I had to declare a separate error variable outside of the scope of the for loop